src/pages/individual/Individual.jsx

```diff
@@ -117,6 +123,38 @@ export default function Individual() {
     },
     [individualData, fieldSchemas],
   );
+  const encounters = get(individualData, 'encounters', []);
+
+  const assetSources = useMemo(
+    () => {
+      const combinedAssets = reduce(
+        encounters,
+        (memo, encounter) => {
+          const annotations = get(encounter, 'annotations', []);
+          const modifiedAssets = map(
+            annotations,
+            (annotation, index) => ({
+              number: index,
+              id: annotation?.asset_guid,
```

Comment:
This key should probably be `guid` instead of `id`.

Reply:
Done